components/widgets/users/users.stories.tsx

```diff
@@ -13,7 +13,7 @@ storiesOf('Users Info', module).
     add(
         'avatars per size',
         () => {
-            const sizes = ['xs', 'sm', 'md', 'lg', 'xl', 'xxl'];
+            const sizes: ('xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl')[] = ['xs', 'sm', 'md', 'lg', 'xl', 'xxl'];
```

Comment:
This can just be `string[]`, it's cleaner and we don't need to be that strict.

Reply:
Here I'm agree, we can use just string[] here (if it works with the restriction at component level), or just let the compiler infer the type.

Reply:
Nop I can't :'(

![Sélection_286](https://user-images.githubusercontent.com/1492516/66079071-a4263080-e563-11e9-893c-675926ae3b60.png)